Library/Formula/libsamplerate.rb

```diff
@@ -26,6 +26,19 @@ def install
     system "./configure", "--disable-dependency-tracking",
                           "--prefix=#{prefix}"
     system "make", "install"
+    rm_f "#{bin}/sndfile-resample" if build.without? "libsndfile"
+  end
+
+  def caveats
+    s = ""
```

Comment:
On the one hand, if the empty string is supported by our code, it probably makes for more readable code to use it because then you can just do `s = ""` and have a sequence of conditionalized appends, without having to guard against `nil` in each of them. And if you really want to return `nil`, the nicest idiom might just be to do that anyway and have the last line be `s == "" ? nil : s` to nil-ify it.

On the other hand, client code which calls `caveats` might prefer to have it be `nil` if it's not populated, so they can then just test for `nil`, or use it as a logical value.

I'm no Ruby expert, but using `nil` to represent "no caveats" seems the cleanest way to do it, because then you have a distinct value with a simple test provided by the language's basic types.